dev/ci/user-overlays/00669-maximedenes-ssr-merge.sh

```diff
@@ -1,3 +1,4 @@
+#!/bin/sh
```

Comment:
I suggest we do not add these shebangs to the files in `user-overlays` as they are not supposed to be executed on their own but sourced somewhere else.

Reply:
Actually, the shebang was added so Shellcheck can check the script is correct according to what is available in a POSIX shell. The shebang won't change anything when sourcing the file, so it's harmless. Moreover, if someone wants to edit the script, it's better to have it as you know it was written as a POSIX shell, which could explain some things in the script. And I don't think someone will try to execute this on its own, as there's no reason to do it.

Anyway, if you still want me to remove all these shebangs, will do. :)

Reply:
It's either remove them or update the README in the same directory.

Reply:
Updated the `README`.